### PR TITLE
Feat: Channel Points Automatic Reward Redemption 

### PR DESCRIPTION
**Pull request by djinnet**
Since the websocket dev branch has been updated to latest changes from dev core in #33 , I do hope it is okay that I am submitted this pull request. Strike while the iron is hot, I suppose. 

Let me know if there are anything that should be modified/changed, so it looks good with the rest of the structure that is already in place.

### CODE REVIEW

File: TwitchLib.EventSub.Websockets/Core/EventArgs/Channel/ChannelPointsAutomaticRewardRedemptionArgs.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿using TwitchLib.EventSub.Core.SubscriptionTypes.Channel;
+using TwitchLib.EventSub.Websockets.Core.Models;
+
+namespace TwitchLib.EventSub.Websockets.Core.EventArgs.Channel;
+public class ChannelPointsAutomaticRewardRedemptionArgs : TwitchLibEventSubEventArgs<EventSubNotification<ChannelPointsAutomaticRewardRedemption>>
```
Comment (AoshiW) [inline TwitchLib.EventSub.Websockets/Core/EventArgs/Channel/ChannelPointsAutomaticRewardRedemptionArgs.cs:5]: missing new line
```suggestion
namespace TwitchLib.EventSub.Websockets.Core.EventArgs.Channel;

public class ChannelPointsAutomaticRewardRedemptionArgs : TwitchLibEventSubEventArgs<EventSubNotification<ChannelPointsAutomaticRewardRedemption>>
```

**Review by AoshiW — CHANGES_REQUESTED** [review]
you forgot to add the handler to `TwitchLib.EventSub.Websockets.Handler.Channel.ChannelPoints.Redemptions`

**Comment by djinnet** [comment]
@AoshiW Thanks for letting me know about the handler. I wasnt aware that it need an handler, since I am not entire familiar with the codebase yet.

I tried to make the handler based of ChannelPointsCustomRewardRedemptionAddHandler.cs, but with automatic rewards. 
Seems good?

**Review by AoshiW — APPROVED** [review]
LGTM

**Comment by djinnet** [comment]
Just out of curiosity, is there more anything that this pull request needs to be tweaked in order to get merged or has it been forgotten to be merged since may? 

I am just asking because last time I touched this pull request was in may and now it is almost at the end of june, so I am little curious.

**Pull request merged**